PendingReleaseNotes

```diff
@@ -99,3 +99,6 @@
 * The RGW api for getting object torrent has changed its params from 'get_torrent'
   to 'torrent' so that it can be compatible with Amazon S3. Now the request for 
   object torrent is like 'GET /ObjectName?torrent'.
+
+* The mon_warn_osd_usage_percent option has been renamed
```

Comment:
The original change is so new I'm not sure it was in any releases to warrant a release note about the name of the option changing.